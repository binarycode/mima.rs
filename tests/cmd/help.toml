args = 'help'
status.code = 0
