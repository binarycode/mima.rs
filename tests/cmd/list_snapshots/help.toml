args = 'help list-snapshots'
status.code = 0
