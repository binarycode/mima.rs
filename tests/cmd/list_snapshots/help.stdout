mima-list-snapshots [VERSION]
List snapshots

USAGE:
    mima list-snapshots <GUEST_ID>

ARGS:
    <GUEST_ID>    Guest ID

OPTIONS:
    -h, --help    Print help information
